src/org/labkey/test/components/react/BaseReactSelect.java

```diff
@@ -255,18 +256,29 @@ public T removeSelection(String value)
 
         scrollIntoView();
 
-        WebElement removeBtn = Locators.removeMultiSelectValueButton(value).findWhenNeeded(getComponentElement());
-        removeBtn.click();
-
-        getWrapper().shortWait().until(ExpectedConditions.stalenessOf(removeBtn));
+        attemptRemove(value);
 
         // Validate that the selected item really was removed.
         WebDriverWrapper.sleep(500);
```

Comment:
It might take a moment for it to reappear (The item definitely goes stale, so it must reappear at some point). I'm not too worried about an extra half second since this has such limited use.